File: 30 Days of Code/Python/Day 8 - Dictionaries and Maps.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
n = int(input())
d = {}
for i in range(0,n):
    s = input().split()
    d[s[0]] =s[1]

while 1:
    try:
        q = input()
        if q in d:
            print(q, "=", d[q], sep="")
        else: 
            print("Not found")
    except:
        break
    

File: 30 Days of Code/PHP/Day 8 - Dictionaries and Maps.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
$n = (int)readline();
$dict = array();
for($i=0;$i<$n;$i++){
    $arr = readline();
    $arr = explode(" ", $arr);
    $dict[$arr[0]] = $arr[1];
}
while(1){
    $q = readline();
    if(!$q)
        break;  

    if(array_key_exists($q, $dict)){
        echo $q."=".$dict[$q]."\n";
    } else {
        echo "Not found\n";
    }
}
?>
